code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm

```diff
@@ -206,13 +225,25 @@
 /obj/item/wheelchair
 	name = "wheelchair"
 	desc = "A folded wheelchair that can be carried around."
-	icon = 'icons/obj/furniture.dmi'
+	icon = 'icons/obj/wheelchair.dmi'
 	icon_state = "wheelchair_folded"
 	item_state = "wheelchair"
 	w_class = ITEMSIZE_HUGE // Can't be put in backpacks. Oh well.
+	var/wheelchairtype = /obj/structure/bed/chair/wheelchair
+	var/wheelchairitemtype = /obj/item/wheelchair
+
+/obj/item/smallmotorwheelchair // need it like this cause the spawning code for wheelchairs in the loadout is funky
+	name = "small electric wheelchair"
+	desc = "A small motorized wheelchair, it looks around the right size for a Teshari."
+	icon = 'icons/obj/wheelchair.dmi'
+	icon_state = "teshchair_folded"
+	item_state = "teshchair"
+	w_class = ITEMSIZE_HUGE // Can't be put in backpacks. Oh well.
+	var/wheelchairtype = /obj/structure/bed/chair/wheelchair/smallmotor
+	var/wheelchairitemtype = /obj/item/smallmotorwheelchair
 
-/obj/item/wheelchair/attack_self(mob/user)
-		var/obj/structure/bed/chair/wheelchair/R = new /obj/structure/bed/chair/wheelchair(user.loc)
+/obj/item/smallmotorwheelchair/attack_self(mob/user)
```

Comment:
Should this path be changed? It looks like you refactored the changes to be on the base wheelchair type.

Reply:
That's being fixed on the next commit